app/helpers/proposals_helper.rb

```diff
@@ -32,4 +32,20 @@ def retire_proposals_options
     Proposal::RETIRE_OPTIONS.collect { |option| [ t("proposals.retire_options.#{option}"), option ] }
   end
 
+  def can_create_document?
+    can?(:create, @document) && @proposal.documents.size < Proposal.max_documents_allowed
```

Comment:
The use of `@document` and `@proposal` at lines 40 and 44 seems strange to me. I mean, this is a helper that share functions across multiple files/views/etc... and this functions assume that the context where they are called will have a variable with that exact name and the right content inside... 

Yes it makes everything quite clean at the view :), but not sure its a good idea from a code point of view (neither I've seen this as a pattern on other helpers of this codebase)

I would go with good old parameters on each function for those 3.